InitializeRepo/initialize.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+git pull > /dev/null
```

Comment:
If you do that, this script will fail here if the working directory isn't a git repository, so if it should work in that case you'll have to wrap this (and all the other git commands) with a check for if this is a working git repo, like:

```sh
# The 2>&1 here prevents git's error text from being shown to the user.

if ! git pull > /dev/null 2>&1; then
    echo "Not in a git repository, couldn't pull"
fi
```

But given that git is used pervasively through the script, we should probably exit instead, which will happen automatically if you use `set -euo pipefail`